code/modules/mob/transform_procs.dm

```diff
@@ -106,7 +106,7 @@
 	new_mob.mutations += M_FAT
 	new_mob.setBrainLoss(100)
 	Postmorph(new_mob)
-	to_chat(new_mob, "<span class='sinister'>Instantly, what was your clothes fall off, and are replaced with a mockery of all that is clowning; Disgusting-looking garb that the foulest of creatures would be afraid of wearing. Your very face begins to shape, mold, into something truely disgusting. A mask made of flesh. Your body is feeling the worst pain it has ever felt. As you think it cannot get any worse, one of your arms turns into a horrific meld of flesh and plastic, making a limb made entirely of bike horns.</span>")
+	to_chat(new_mob, "<span class='sinister'>Instantly, what was your clothes fall off, and are replaced with a mockery of all that is clowning; A disgusting-looking garb that the foulest of creatures would be afraid of wearing. Your very face begins to shape, mold, into something truly disgusting. A mask made of flesh. Your body is feeling the worst pain it has ever felt. As you think it cannot get any worse, one of your arms turns into a horrific meld of flesh and plastic, making a limb made entirely of bike horns.</span>")
```

Comment:
A lack of article is actually correct here because this is being used in the uncountable sense.

"He wore disgusting garb."